src/main/java/be/syntra/devshop/DevshopFront/services/PaymentServiceImpl.java

```diff
@@ -0,0 +1,34 @@
+package be.syntra.devshop.DevshopFront.services;
+
+import be.syntra.devshop.DevshopFront.models.StatusNotification;
+import org.springframework.stereotype.Service;
+
+@Service
+public class PaymentServiceImpl implements PaymentService {
+//
+//    @Value("${baseUrl}")
+//    private String baseUrl;
+//
+//    @Value("${cartEndpoint}")
+//    private String endpoint;
+//
+//    private String resourceUrl = null;
+//
+//    @Autowired
+//    private RestTemplate restTemplate;
+//
```

Comment:
not part of DEV-012. Please remove. If removal proves to be difficult because of the Controller implementation, have the controller receive null directly and still remove this class